paths_cli/wizard/shooting.py

```diff
@@ -0,0 +1,128 @@
+from functools import partial
+
+from paths_cli.wizard.core import get_missing_object
+from paths_cli.wizard.engines import engines
+from paths_cli.wizard.cvs import cvs
+from paths_cli.parsing.tools import custom_eval
+
+import numpy as np
+
+
+def uniform_selector(wizard):
+    import openpathsampling as paths
+    return paths.UniformSelector()
+
+def gaussian_selector(wizard):
+    import openpathsampling as paths
+    cv_name = wizard.ask_enumerate("Which CV do you want the Gaussian to "
+                                   "be based on?",
+                                   options=wizard.cvs.keys())
+    cv = wizard.cvs[cv_name]
+    l_0 = wizard.ask_custom_eval(f"At what value of {cv.name} should the "
+                                 "Gaussian be centered?")
+    std = wizard.ask_custom_eval("What should be the standard deviation of "
+                                 "the Gaussian?")
+    alpha = 0.5 / (std**2)
+    selector = paths.GaussianBiasSelector(cv, alpha=alpha, l_0=l_0)
+    return selector
+
+# def random_velocities(wizard):
+    # pass
+
+# def gaussian_momentum_shift(wizard):
+    # pass
+
+# def get_allowed_modifiers(engine):
+    # allowed = []
+    # randomize_attribs = ['randomize_velocities', 'apply_constraints']
+    # if any(hasattr(engine, attr) for attr in randomize_attribs):
+        # allowed.append('random_velocities')
+
+    # if not engine.has_constraints():
+        # allowed.append('velocity_changers')
+
+    # return allowed
```

Comment:
are these functions commented out by design?

Reply:
Yes, these are left as commented code as a reminder to implement more later. The first version of the Wizard will be relatively limited (since the PR is already huge), but later extension should be straightforward.